AltinnReStorage/AltinnReStorage.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Attributes\StephAttribute.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="2.2.2" />
+    <PackageReference Include="Azure.Identity" Version="1.1.1" />
+    <PackageReference Include="Azure.ResourceManager.Storage" Version="1.0.0-preview.1" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.4.4" />
```

Comment:
12.5.0 is latest. Seems to contain some important fixes.
https://github.com/Azure/azure-sdk-for-net/blob/Azure.Storage.Blobs_12.5.0/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

Reply:
:+1: